search-sentiments-map/routes/search.js

```diff
@@ -24,6 +25,8 @@ const datastore = new Datastore();
 const json = require('./../public/country-code.json');
 global.Headers = fetch.Headers;
 
+const SEVENDAYS = 7 * 24 * 60 * 60000;
```

Comment:
For anything to do with time, we typicallydo the following:
- start with what this number is for
- duration
- units

Also, underscores to separate words.

So this would look something like:
STALE_SEARCH_RESULT_THRESHOLD_7_DAYS_MS